mezzanine/pages/static/mezzanine/js/admin/page_tree.js

```diff
@@ -59,21 +59,13 @@ jQuery(function($) {
         });
     }
 
-    // The dropdown list for adding a new page contains URLs for each
-    // model - redirect when selected.
     $('.addlist').change(function() {
         // Ensure the branch is saved as open when adding to it so that
         // the new branch will be visible directly after saving.
         var id = $(this).attr('id');
         if (id) {
             toggleID(true, id.split('-')[1]);
         }
-        var addUrl = this[this.selectedIndex].value;
-        if (addUrl) {
-            location.href = addUrl;
-        }
-        this.selectedIndex = 0;
```

Comment:
It's only a guess but it's a confident one - I think that was to handle the situation when the user had been directed to the "add" interface, hit the back button, and the browser had the tree page in cache/memory, and keeps the state of the select list. So the select list would have a page type selected, and there'd be no way to trigger the JS to run again other than manually selecting nothing (eg selectedIndex = 0) and selecting it again.